staging/src/k8s.io/client-go/util/flowcontrol/throttle.go

```diff
@@ -23,26 +23,36 @@ import (
 	"time"
 
 	"golang.org/x/time/rate"
+	"k8s.io/utils/clock"
 )
 
-type RateLimiter interface {
+type PassiveRateLimiter interface {
```

Comment:
what's the difference between PassiveRateLimiter and RateLimiter? it's not clear why TryAccept belongs in once interface and Accept belongs in the other

Reply:
PassiveRateLimiter reads time but does do any waiting.